### ISSUE TITLE
Compile war not loading assets

### ISSUE THREAD

**Issue by enyachoke** [jruby/warbler#169]
I have compiled a jruby rails4 app. Everthing runs fine except css and js files are not being served. Looking at th error logs I see.This

ActionController::RoutingError (No route matches [GET] "/stylesheets/application.css"):|  actionpack (4.0.0) lib/action_dispatch/middleware/debug_exceptions.rb:21:in `call'|  actionpack (4.0.0) lib/action_dispatch/middleware/show_exceptions.rb:30:in`call'|  railties (4.0.0) lib/rails/rack/logger.rb:38:in `call_app'|  railties (4.0.0) lib/rails/rack/logger.rb:21:in`call'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `tagged'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:25:in`tagged'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in `tagged'|  railties (4.0.0) lib/rails/rack/logger.rb:21:in`call'|  actionpack (4.0.0) lib/action_dispatch/middleware/request_id.rb:21:in `call'|  rack (1.5.2) lib/rack/methodoverride.rb:21:in`call'|  rack (1.5.2) lib/rack/runtime.rb:17:in `call'|  activesupport (4.0.0) lib/active_support/cache/strategy/local_cache.rb:83:in`call'|  railties (4.0.0) lib/rails/engine.rb:511:in `call'|  railties (4.0.0) lib/rails/application.rb:97:in`call'|  file:lib/jruby-rack-1.1.13.2.jar!/rack/handler/servlet.rb:22:in `call'|||
2013-07-08 18:57:53.784:INFO:/:Started GET "/javascripts/application.js" for 127.0.0.1 at 2013-07-08 18:57:53 +0300|
2013-07-08 18:57:53.949:INFO:/:|ActionController::RoutingError (No route matches [GET] "/javascripts/application.js"):|  actionpack (4.0.0) lib/action_dispatch/middleware/debug_exceptions.rb:21:in`call'|  actionpack (4.0.0) lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'|  railties (4.0.0) lib/rails/rack/logger.rb:38:in`call_app'|  railties (4.0.0) lib/rails/rack/logger.rb:21:in `call'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in`tagged'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:25:in `tagged'|  activesupport (4.0.0) lib/active_support/tagged_logging.rb:67:in`tagged'|  railties (4.0.0) lib/rails/rack/logger.rb:21:in `call'|  actionpack (4.0.0) lib/action_dispatch/middleware/request_id.rb:21:in`call'|  rack (1.5.2) lib/rack/methodoverride.rb:21:in `call'|  rack (1.5.2) lib/rack/runtime.rb:17:in`call'|  activesupport (4.0.0) lib/active_support/cache/strategy/local_cache.rb:83:in `call'|  railties (4.0.0) lib/rails/engine.rb:511:in`call'|  railties (4.0.0) lib/rails/application.rb:97:in `call'|  file:lib/jruby-rack-1.1.13.2.jar!/rack/handler/servlet.rb:22:in`call'|||

Any Ideas. Please
I use this in my warble.rb
config.features = %w(runnable compiled executable)

**Comment by ctreatma** [comment]
This problem was reintroduced by rails/sprockets#7, which changed the default manifest file from `manifest-<md5>.json` to `.sprockets-manifest-<md5>.json`.

**Comment by ctreatma** [comment]
I've been looking into this more, and it seems like warbler 1.4.2+ does not pull in the asset manifest when assets are precompiled by hooking into the `war` rake task as documented in #57.  I created the [ctreatma/warbler_sprockets_3](https://github.com/ctreatma/warbler_sprockets_3) repo to demonstrate the issue; I have managed to reproduce it, but am still working to find the root cause.  It appears that, starting with 1.4.2,

    config.includes += FileList["public/assets/.sprockets-manifest-*.json"].existing

is evaluated before the `assets:precompile` task runs, so it does not find a manifest to include in the WAR.
